setup_dependencies.org

```diff
@@ -33,8 +33,16 @@ https://github.com/libbpf/libbpf.
 
 ** libxdp and libbpf as git-submodules
 
-This repository uses both [[https://github.com/xdp-project/xdp-tools/][libxdp]] and [[https://github.com/libbpf/libbpf][libbpf]] as git-submodules. After cloning
-this repository you need to run the command:
+This repository uses both [[https://github.com/xdp-project/xdp-tools/][libxdp]] and [[https://github.com/libbpf/libbpf][libbpf]] as git-submodules.
+
+We will start by cloning this repository:
+
+#+begin_example
+git clone git@github.com:xdp-project/xdp-tutorial.git
```

Comment:
We use the https URL everywhere else; let's do the same here...

Reply:
Also, if we're adding this, I think it's better to add the `--recurse-submodules` switch here (and get rid of it below), then rewrite the text below to explain that the `git submodule update` command is only needed if `--recurse-submodules` was not used in the initial clone command...